.gitattributes

```diff
@@ -1,35 +1,36 @@
-# NOTE: this file is for git 1.6.6 (and possibly older)
-# Post-1.7.2 there is eol and text, and crlf is deprecated
-# but we can't depend on an unreleased version...
+# Set default behavior to automatically normalize line endings.
+* text=auto
 
-# This file mainly controls line ending conversion behaviour, if
-# the user has the setting core.autocrlf true.
+# This file mainly controls line ending conversion behavior.
+# Example: https://github.com/dotnet/corefx/blob/e1eb5618df4e06a9dd9e44db349fb20d14f8d3ac/.gitattributes
+# Docs:    https://git-scm.com/docs/gitattributes
 
 # The meaning of the attributes is a little odd
-# -crlf means DO NOT convert line endings
-# crlf means CONVERT to lf in the repo & Linux/Mac, crlf on Windows
+# crlf means CONVERT to lf in the repo & Linux/Mac, checkout as crlf on Windows
```

Comment:
Wait, what?  This doesn't make sense to me.  Consider `Xamarin.Android.sln`, which is currently `*.sln eol=crlf`.  On my macOS checkout, it is in fact CRLF:

```bash
$ hexdump -C Xamarin.Android.sln
...
00000040  0d 0a 23 20 56 69 73 75  61 6c 20 53 74 75 64 69  |..# Visual Studi|
```

Note `0d 0a` -- CRLF.

If `*.sln text eol=crlf` means "convert to lf on Linux/Mac", then now my checkout will be different (dirty?).

That also sounds potentially "wrong"; what will Visual Studio for Mac do with a LF-formatted `.sln` file?  (I have  no idea, and am not sure I want to find out.)